src/ui-actions/participant-displayer-actions.ts

```diff
@@ -23,5 +23,6 @@ export const copyEmailToClipboard = (email: string, createSnackbar: CreateSnackb
 };
 
 export const sendMsg = (contact: Participant): void => {
-	mailToSharedFunction([contact]);
+	const updatedContact: Participant = { ...contact, type: 't' };
```

Comment:
Instead of the _'t'_ string I suggest to use the constants from `ParticipantRole` enumeration